Minor fixes to smooth devops experience

sampaccoud opened the pull request:
## Purpose

I and an external user bumped into small glitches while trying to deploy Open edX projects. Let's smooth things out.


## Proposal

- [x] We only want to add the project display name with the "env_type" between brackets for the name of the project in OpenShift. In edX, the PLATFORM_NAME is used for example in buttons and should therefore be a short display name.
- [x] Add a bin script to build images as it was recently removed from the deploy process,
- [x] Rename the `preproduction` environment to preprod for coherence with fun-platform.

sampaccoud committed:
♻️(env_type) rename the preproduction env_type to preprod

This environment is called preprod in fun-platform so we should
call it by the same name for coherence. Preprod is shorter and
frequently used.

sampaccoud committed:
🚸(bin) add script to build images as it is now separate from deploy

We recently took the image build out of deployment because we were
making many more deployments than image builds and the image build
is consuming a lot of resources. We forgot to add a helper for this
command forcing users to either use bin/bootstrap or type the whole
docker run command.

jmaupetit commented on group_vars/customer/campus/main.yml:
```diff
@@ -1,6 +1,6 @@
-project_display_name: "FUN Campus ({{ env_type }})"
-project_description: "SPOC for universities ({{ env_type }})"
+project_display_name: "FUN Campus"
+project_description: "SPOC for universities"
```
IMO there is a confusion between the OpenShift and the application layer. In an OpenShift context I want to get the `env_type`. I think, this should be fixed in the application layer (set the `SITE_NAME` to `{{ customer }}` for example). Do you read me?

jmaupetit commented on bin/build_images:
```diff
@@ -0,0 +1,11 @@
+_set_openshift_env
+
+# Run ansible-playbook
+_docker_run ansible-playbook build_images.yml "$@"
```
:medal_sports: I've missed this one.

jmaupetit replied:
FYI you are not forced to use the whole docker command, I used the `bin/ansible-playbook` helper.

sampaccoud committed:
🚸(edxapp) improve PLATFORM_NAME setting

In Open edX, the PLATFORM_NAME setting is used in buttons and other
places that require a short display name. The project display name is
for use in OpenShift and contains information about the environment
that is not wanted in the application.
